controllers/admin/AdminImagesController.php

```diff
@@ -178,6 +192,51 @@ public function __construct()
                 ],
                 'submit' => ['title' => $this->trans('Save', [], 'Admin.Actions')],
             ],
+            'optimization' => [
+                'title' => $this->trans('Advanced optimization options', [], 'Admin.Design.Feature'),
+                'icon' => 'icon-picture',
+                'top' => '',
+                'bottom' => '',
+                'description' => $this->trans('Optimize bandwidth and disk usage.', [], 'Admin.Design.Help'),
+                'fields' => [
+                    'PS_IMAGEOPT_NO_ENLARGE' => [
+                        'type' => 'bool',
+                        'title' => $this->trans('Never create images larger than the base image', [], 'Admin.Design.Feature'),
```

Comment:
```suggestion
                        'title' => $this->trans('Always keep images smaller than the base image', [], 'Admin.Design.Feature'),
```

I tried to use a positive sentence instead.

Reply:
That's confusing in my opinion. I suggest `Do not upscale images` or keep @lmeyer1's suggestion.

The hint should be something like: 'If the thumbnail is larger than the base image, keep the original size and do not upscale it.'

Reply:
@Julievrz
I kept @Hlavtox's suggestion.